mobile/lib/flashcard/show/screens/flashcard_page.dart

```diff
@@ -21,49 +25,65 @@ class FlashcardPage extends StatefulWidget {
 class _FlashcardPageState extends State<FlashcardPage> {
   String question = "Question";
   String answer = "Answer";
+  bool isFavorite = false;
 
-  Future<FlashcardDTO?> _getFlashcard(BuildContext context) async {
-    int apiId = context.read<ItemListProvider>().filteredItems.results[widget.id].id;
-    FlashcardDetailsResponse? response =
-        await FlashcardEndpointApi(apiClient).getById(apiId);
-    if (response != null) {
-      Map<String, dynamic> json = jsonDecode(jsonEncode(response));
-      return FlashcardDTO.fromJSON(json);
-    }
-    return null;
+  @override
+  void initState() {
+    super.initState();
+    context.read<ItemListProvider>().clearProvider();
```

Comment:
jaki jest sens czyszczenia providera i pobierania ulubionych za każdym odpaleniem fiszki? to powinno działać tylko w pokazywaniu ulubionych. Powinna być funkcja do ustawiania/pobierania czy jest ulubiona fiszka.